test/specs/home.spec.js

```diff
@@ -11,8 +13,35 @@ describe('Wikipedia home page',  () => {
     })
 
     it('should allow the user to search and display results', () => {
-        homePage.setSearchText("world's tallest building")
-        // This pause is here so you can observe the test state with npm run test:head
-        browser.pause(10000)
+        homePage.setSearchText(searchKey)
+        expect(homePage.suggestionsResult(searchKey).waitForDisplayed()).toBeTruthy()
+    })
+
+    it('should navigate to search result when clicked', () => {
+        homePage.suggestionsResult(searchKey).click()
+        homePage.waitForPageLoad()
+        const expected = "https://en.wikipedia.org/wiki/List_of_tallest_buildings"
+        const pageUrl = browser.getUrl()
+        expect(pageUrl).toEqual(expected)
     })
+
+    it('should find second tallest building and verify height is correctly converted', () => {
+        expect(homePage.secondTallestName.getText()).toEqual("Shanghai Tower")
+        expect(homePage.secondTallestHeightFt.getText()).toEqual('2,073\xA0ft')
```

Comment:
Could you actually run the math to make sure the conversion is correct?

Reply:
Done, made sure conversion is actually working as expected.

Reply:
Thank you much! Just what I was hoping for.